### PR TITLE
BAQE-1474: Fix random failures when running clustered tests using Ope…

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/BAQE-1474
This ticket addresses two random failures:
- Space is not synced yet after being created
- The rollout already happened when disposing containers.

### CODE REVIEW

File: test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/client/util/KieServerUtils.java [hunk 1]
```diff
@@ -51,6 +62,34 @@
...
+     * @param kieServer
+     * @param containerId
+     */
+    public static void waitForContainerRespinAfterDisposeContainer(KieServerDeployment kieServer, String containerId) {
+        KieServicesClient kieServerClient = KieServerClientProvider.getKieServerClient(kieServer);
+        waitForContainerRespinAfter(kieServer, () -> kieServerClient.disposeContainer(containerId));
+    }
+
+    /**
+     * Wait for container respoin after doing an action.
```
Comment (jakubschwan) [inline test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/client/util/KieServerUtils.java:76]:
```suggestion
     * Wait for container respin after doing an action.
```

File: test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/curl/CurlCommand.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+     * @return
+     */
+    public String get(String path) {
+        return runCommand(GET_METHOD, path);
+    }
+
+    private String runCommand(String method, String path) {
+        String command = String.format("curl -u %s:%s -X %s http://%s:%s/%s", username, password, method, host, port, path);
+        LOGGER.debug("Running command in {}: {}", pod.getName(), command);
+        CommandExecutionResult result = pod.runCommand();
```
Comment (jakubschwan) [inline test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/curl/CurlCommand.java:95]: Isn't there missing command to be run?
```suggestion
        CommandExecutionResult result = pod.runCommand(command);
```

Comment (Sgitario) [reply test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/curl/CurlCommand.java:95]: Well spotted. I'm checking why the test is working fine...

Comment (Sgitario) [reply test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/curl/CurlCommand.java]: It seems that the TimeUtils didn't finish the exec if the condition was not met. I've already corrected this also.

**Comment by jakubschwan** [comment]
@Sgitario there is compilation failure in CommandUtil, for details please take a look into failing Jenkins job log

**Comment by Sgitario** [comment]
> @Sgitario there is compilation failure in CommandUtil, for details please take a look into failing Jenkins job log

Fixed by using Java 8.

**Pull request merged**